### PR TITLE
task4-all articles can be viewed on the home page

### PR DESCRIPTION
**Pull request by Hanieh73**
task4-all articles can be viewed on the home page.

### CODE REVIEW

**Comment by Harrison365** [comment]
Hi Hanieh :) 

Currently when you try to load your front end it gives a page not found error and you have to go to /articles to see them.
To fix this, either put a little link on the home page to the articles or change the endpoint for your articles component to "/".

You will also need to make sure that your components are in separate files. Your App.js is starting to get a little crowded. 

Please also add isLoading functionality for your articles list.

If you have any issues with these, please let me know 🙂 
Send us a new nchelp pr, when you are ready for us to take another look 👍

File: nc-news/src/Components/ArticleList.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { fetchAllArticles } from "../utils";
+import { fetchArticleById } from "../utils";
+import ArticleCard from "./ArticleCard";
+
+
+const ArticleList = (
+    { article, allArticles, setArticle, setAllArticles, allTopics, isLoading, setIsLoading
+    }) => {
+    //clarify
+    const resolvedReset = () => {
```
Comment (lewispricey) [inline nc-news/src/Components/ArticleList.jsx:11]: This function doesn't ever look to be used

Comment (Hanieh73) [reply nc-news/src/Components/ArticleList.jsx]: the unused function has been removed now

File: nc-news/src/App.js [hunk 2]
```diff
@@ -1,14 +1,105 @@
...
+import {Routes, Route} from 'react-router-dom'
+import { useEffect, useState } from 'react';
+import {fetchAllArticles} from './utils'
 
 
 
 
 function App() {
+  const [allTopics, setAllTopics] = useState([])
+  const [allArticles, setAllArticles] = useState([])
```
[the remark this answers is not in the document]
Comment (Hanieh73) [reply nc-news/src/App.js]: some states have now been moved down the tree, from the App.js

**Pull request merged**